spec/features/user_contributes_new_councillor_spec.rb

```diff
@@ -66,5 +66,63 @@
 
       expect(page).to have_content "Thank you"
     end
+
+    it "successfully submit suggested councillor(s) information with a blank field of add another councillors" do
```

Comment:
Because the feature is "Contributing a new councillor for an authority" I think we can drop the 'submit ...' from this, to make it just "successfully with 3 completed and one blank councillor". That's a more detailed and more focused description.

For the final test maybe "successfully with three councillors".

With these tests descriptions make a change to get it to fail and see what the message is on the command line. It should be a clear, short sentence if possible, e.g. "Contributing councillors for an authority successfully with three councillors". The 'with three councillors' might be redundant at this point, but because we've got one with a blank let's keep the numbers to make it clear whats going on for now I recon.